src/api/app/policies/attrib_policy.rb

```diff
@@ -3,11 +3,14 @@ def create?
     # Admins can write everything
     return true if @user.is_admin?
 
-    # No specific rules set for the attribute, check if the user can modify the container
-    return true if @record.container.try(:can_be_modified_by?, @user)
-
-    # check for modifiable_by rules
-    permissions_for_modifiables?(modifiables)
+    at_modifiables = modifiables
```

Comment:
That's a good idea. But for now I want to get the regression fixed asap.

I'll get back to this one when the other sprint card got fixed. And then I am going to try your suggestion:-)